packaging/brew/ice.rb

```diff
@@ -0,0 +1,77 @@
+class Ice < Formula
+    desc "Comprehensive RPC framework"
+    homepage "https://zeroc.com"
+
+    version "${ICE_VERSION}"
+    url "${ICE_URL}"
+    sha256 "${ICE_URL_SHA256}"
+
+    depends_on "lmdb"
+    depends_on "mcpp"
+
+    def install
+      args = [
+        "prefix=#{prefix}",
+        "V=1",
+        "USR_DIR_INSTALL=yes", # ensure slice and man files are installed to share
+        "MCPP_HOME=#{Formula["mcpp"].opt_prefix}",
+        "LMDB_HOME=#{Formula["lmdb"].opt_prefix}",
+        "CONFIGS=all",
+        "PLATFORMS=all",
+        "LANGUAGES=cpp",
+      ]
+      system "make", "install", *args
+
+      (libexec/"bin").mkpath
+      %w[slice2py slice2rb slice2js].each do |r|
```

Comment:
Can we remove those? For these the compiler are in the respective pip/gem/npm package.

Reply:
I guess so.